source/_integrations/homeassistant_hardware.markdown

```diff
@@ -14,7 +14,7 @@ ha_platforms:
 ---
 
 The Home Assistant Hardware integration provides functionality
-shared between Home Assistant Yellow and Home Assistant Connect ZBT-1.
+shared between [Home Assistant Yellow](/integrations/homeassistant_yellow/), [Home Assistant Connect ZBT-1](/integrations/homeassistant_sky_connect/), and [Home Assistant Green](/integrations/homeassistant_green).
```

Comment:
Side note: Maybe order the products 🔡 unless there's a purpose to the current order?

Reply:
oh, yes. Good point. Thank you for the catch.